### ISSUE TITLE
Assignment 22

### ISSUE THREAD

**Issue by twhitacre** [tiy-atl-js-q2-2015/Assignments#414]

**Comment by MeadeSlocomb** [comment]
Tim, I may have been a bit too ambitious on this one.  It looks like I'm going to need your assistance at some point to finish this up.

The idea was to make an app that allows a user to submit a set of trivia questions, answers and a time limit for the quiz.  Users can then browse a list of quizzes, select and play them.  In hindsight, this was probably too much for a 2 day project and only 2 days of angular under my belt.  Here's what I accomplished:

Users:

Users can sign up and log in.  Parse will authenticate the log in and an auth token will be stored in a cookie.  I did not have enough time to add a welcome message or sign out feature.  I also did not restrict any of the site functions to logged-in users.

Adding quizzes:

Users can create a quiz with a title, description, and  unlimited number of questions.  I did not get to the time limit feature.

Playing quizzes:  Users can browse and choose a quiz from the main page which will route to a quiz playing page.  Users can see the questions and answer them, but I am not yet able to store those answers properly so that they can be checked against the correct answers, this is mainly what I need help with.

Styling: Though not as bad as some of my previous projects, the styling is unfinished.  I'm hoping to add the name of the app next to the logo, and allow quiz creators to choose a quiz subject which will correspond to an icon that is displayed next to the quiz on the main page at the very least.


Sorry for submitting  an essay along with my project, just felt I needed to explain what you are looking at.  I went ahead and deployed what I have so far:

http://development.meade-popquiz.divshot.io/#/addquiz
https://github.com/MeadeSlocomb/PopQuiz
